.travis.yml

```diff
@@ -0,0 +1,44 @@
+sudo: true
+language: python
+python:
+  - "2.7"
+addons:
+  apt:
+    sources:
+    - precise-pgdg-9.5
+    packages:
+    - postgresql-9.5
+    - postgresql-contrib-9.5
+  postgresql: 9.5
+addons:
+  postgresql: "9.5"
+before_install:
+  - sudo /etc/init.d/postgresql stop
+  - sudo apt-get -y remove --purge postgresql-9.1
+  - sudo apt-get -y remove --purge postgresql-9.2
+  - sudo apt-get -y remove --purge postgresql-9.3
+  - sudo apt-get -y remove --purge postgresql-9.4
+  - sudo apt-get -y autoremove
+  - sudo apt-key adv --keyserver keys.gnupg.net --recv-keys 7FCC7D46ACCC4CF8
+  - sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ precise-pgdg main 9.5" >> /etc/apt/sources.list.d/postgresql.list'
+  - sudo apt-get update
+  - sudo apt-get -y install postgresql-9.5
+  - sudo sh -c 'echo "local all postgres trust" > /etc/postgresql/9.5/main/pg_hba.conf'
+  - sudo sh -c 'echo -n "host all all 127.0.0.1/32 trust" >> /etc/postgresql/9.5/main/pg_hba.conf'
+  - sudo /etc/init.d/postgresql restart
+  - sudo -u postgres createuser test_user
+  - psql --version
+
+install:
+  - pip install --process-dependency-links -e .
+  - cp portal/application.cfg.travis portal/application.cfg
+script: nosetests -v --attr '!travis_fail'
+before_script:
+ - psql -c 'create database travis_ci_test;' -U postgres
+ - psql -c 'create database portal_unit_tests;' -U postgres
+ - python manage.py initdb
+ - python manage.py db stamp 24d8d4008534
```

Comment:
Grab the current head for the stamp call:
python manage.py db stamp `python manage.py db heads | awk '{print $1}'`